csrankings-7.csv

```diff
@@ -1878,7 +1878,7 @@ Sanjiang Li,University of Technology Sydney,https://www.uts.edu.au/staff/sanjian
 Sanjib Sur,University of South Carolina,https://cse.sc.edu/~sur,sgShf0sAAAAJ
 Sanjit A. Seshia,Univ. of California - Berkeley,https://people.eecs.berkeley.edu/~sseshia,SlZavnIAAAAJ
 Sanjit Chatterjee,IISc Bangalore,http://www.csa.iisc.ac.in/~sanjit,bbLLOyIAAAAJ
-Sanjiv Kapoor,Illinois Institute of Technology,https://science.iit.edu/people/faculty/sanjiv-kapoor,NOSCHOLARPAGE
+Sanjiv Kapoor,Illinois Institute of Technology,https://science.iit.edu/people/faculty/sanjiv-kapoor,5KwApbgAAAAJstefan

```

Comment:
spuriously says "stefan" at the end, please fix.

Reply:
Removed "stefan" from the patch.